src/CMakeLists.txt

```diff
@@ -698,9 +698,15 @@ if(WITH_STATIC_LIBSTDCXX)
   target_link_libraries(common
     INTERFACE "-static-libstdc++ -static-libgcc")
 endif()
+if(HAVE_QATZIP)
+  target_link_libraries(common PRIVATE ${QATZIP_LIBRARIES})
+endif()
 
 add_library(ceph-common SHARED ${ceph_common_objs})
 target_link_libraries(ceph-common ${ceph_common_deps})
+if(HAVE_QATZIP)
```

Comment:
can we add `${QATZIP_LIBRARIES}` to `ceph_common_deps` to avoid the duplication?

Reply:
Yes. it should be so.